wct.conf.js

```diff
@@ -0,0 +1,7 @@
+const packages = require( './package.json' );
```

Comment:
Name  `packages` is a bit confusing (espetialy later on the `packages.packages` part) - please consider some better name:
- package
- packageJson
- packageInfo